ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/WorkloadCvesPage.tsx

```diff
@@ -20,9 +23,56 @@ import NamespaceViewPage from './NamespaceView/NamespaceViewPage';
 import { WorkloadCveViewContext } from './WorkloadCveViewContext';
 
 import './WorkloadCvesPage.css';
+import { QuerySearchFilter } from '../types';
+
+export const userWorkloadViewId = 'user-workloads';
+export const platformViewId = 'platform';
+export const allImagesViewId = 'all-images';
+export const inactiveImagesViewId = 'inactive-images';
```

Comment:
View id is consistent with path segment and route key? It rhymes :)
Intuitive, yet forward-thinking to define the values independently.